### DISCUSSION TITLE
Use template to generate entity name?

### DISCUSSION THREAD

**Discussion by Qwarkk6** [kalkih/mini-graph-card#832] [Q&A]
Is there any way to use a template to determine the name given to an entity?

Any attempt I make just returns [object object]. 

Thank you.

**Comment by ildar170975** [comment]
Templates are not supported. 
You may place the whole card inside config-template-card.

**Reply by Qwarkk6** [reply]
Thank you for your suggestion. The documentation for that custom card just isn't detailed enough for me to use.

I gave it a try but i'm not convinced it allows for timestamp templating.

**Reply by ildar170975** [reply]
The config-template-card allows templates based on JS. 
See GitHub for examples. 
Any questions may be asked in a dedicated config-template-card community thread.
